main/dbus/APKBUILD

```diff
@@ -3,16 +3,17 @@ pkgname=dbus
 pkgver=1.12.16
 pkgrel=0
 pkgdesc="Freedesktop.org message bus system"
+options="!check" # Introduces circular dependency with xorg-server (xvfb-run -> xvfb)
 url="https://www.freedesktop.org/Software/dbus"
 pkggroups="messagebus"
 arch="all"
-options="!check"  # Requires running X11 server.
 license="AFL-2.1 OR GPL-2.0-or-later"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-x11 $pkgname-openrc"
 triggers="$pkgname.trigger=/etc/dbus-1/system.d"
 depends_dev="util-linux-dev"
 makedepends="$depends_dev expat-dev libx11-dev autoconf automake libtool xmlto
-	libsm-dev"
+	libsm-dev glib-dev"
+checkdepends="xvfb-run"
```

Comment:
```suggestion
#checkdepends="xvfb-run"
```

Reply:
Check is disabled, checkdepends isn't taken into account.

Reply:
You're correct. `want_check` prevents this variable from being used.